src/libs/actions/FormActions.js

```diff
@@ -16,6 +16,14 @@ function setErrorMessage(formID, error) {
     Onyx.merge(formID, {error});
 }
 
+/**
+ * @param {String} formID
+ * @param {String} errors
+ */
+function setErrors(formID, errors) {
+    Onyx.merge(formID, {errors});
+}
+
```

Comment:
Why did we create this? I think that we should update the existing `setErrorMessage` function to conform with the new format we have for errors. We are still early on in the form refactors so it shouldn't be hard to update other instances.

Reply:
Got it. I will work on it.

Reply:
done @luacmartins. Also updated my PR [here](https://github.com/Expensify/App/pull/11076) to use `errors`.